### ISSUE TITLE
[BUG] Unicode characters not supported by base OS X are used in web client tabs

### ISSUE THREAD

**Issue by luciensadi** [evennia/evennia#2181]
#### Describe the bug
On Mac OS X, the web client's usage of these character codes is unsupported:

```
./evennia/web/webclient/static/webclient/js/plugins/goldenlayout.js:287:        let renameDropdownControl = $("<span class='lm_title' style='font-size: 1.5em;width: 0.5em;'>&#129170;</span>");
./evennia/web/webclient/static/webclient/js/plugins/goldenlayout.js:288:        let typeDropdownControl   = $("<span class='lm_title' style='font-size: 1.0em;width: 1em;'>&#11201;</span>");
./evennia/web/webclient/static/webclient/js/plugins/goldenlayout.js:289:        let updateDropdownControl = $("<span class='lm_title' style='font-size: 1.0em;width: 1em;'>&#11208;</span>");
```

Instead of displaying as actual characters, they display as number-boxes in Firefox and empty-boxes in Chrome.

![erroneous display in Firefox](https://user-images.githubusercontent.com/28811446/89717159-90f4a600-d968-11ea-99e6-4fefe995d70e.png)
![erroneous display in Chrome](https://user-images.githubusercontent.com/28811446/89717162-96ea8700-d968-11ea-8caf-e799eed48e5c.png)

#### To Reproduce
1. Open the webclient on OS X.

#### Expected behavior
Supported characters should be used. I'm not sure what the expected display is, so I can't offer any suggestions at this time.

#### Environment, Evennia version, OS etc
```
    Evennia 0.9.0 (rev c3831ea09) (rev c3831ea09)
    OS: posix
    Python: 3.8.5
    Twisted: 20.3.0
    Django: 2.2.15
```

**Comment by Griatch** [comment]
Sounds like something that we should move to a separate css file and add more fallback options for the font. (cc @friarzen for awareness)

**Comment by Griatch** [comment]
@friarzen Is this something you'd have time to take a peek at?

**Comment by Griatch** [comment]
Closed in #2901.
